### PR TITLE
Configuration is updated to use files or environment variables.

### PR DESCRIPTION
**Pull request by Skarlso**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -83,6 +83,45 @@ Gaia will automatically detect the folder of the binary and will place all data
...
+
+2. Environment Properties
+
+Environment variables can be used to overwrite any configuration settings / defaults that are already defined in Gaia.
+
+All Environment variables must be pre-fixed with `GAIA_` in order to avoid collision with other variables.
+
+For example:
+
+.. code:: bash
```
Comment (Skarlso) [inline README.rst]: @michelvocks How do I format code? :O This isn't showing up. :O

**Comment by Skarlso** [comment]
Deals with #32

File: README.rst (hunk 1 above)
Comment (michelvocks) [reply README.rst]: You need to add an empty line after `.. code:: bash` for example:

.. code: bash

    #!/bin/bash
    echo hello world

Comment (Skarlso) [reply README.rst]: Ah, thanks 🙇 :)

**Comment by michelvocks** [comment]
What do you think about moving the documentation part out of the README.rst file and into the [docs](https://docs.gaia-pipeline.io/)? 😉 
This is information is important but I'm not sure if it is really necessary to add this to the README.rst file? Otherwise LGTM 🤗

**Comment by Skarlso** [comment]
@michelvocks Hum. Where do you think I should put it? Should I create a new section? 
Should I add it here? https://github.com/gaia-pipeline/docs/blob/master/content/getting-started/install-gaia.md

**Comment by michelvocks** [comment]
https://github.com/gaia-pipeline/docs/blob/master/content/getting-started/install-gaia.md is fine I guess. 😄

**Pull request merged**